app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-schema.vue

```diff
@@ -260,9 +260,7 @@ export default defineComponent({
 
 		const typesWithLabels = computed(() => translate(fieldTypes));
 
-		const typeDisabled = computed(() => {
-			return ['file', 'files', 'o2m', 'm2m', 'm2a', 'm2o', 'translations'].includes(localType.value);
-		});
+		const typeDisabled = computed(() => localType.value !== 'standard');
```

Comment:
This is not the same. You're no longer checking for hash / uuid / json / csv

Reply:
The `localType` can only be one of those, so the lines should be equivalent ?
https://github.com/directus/directus/blob/5feaac88d7edf68861a050c7c04cd3ab50a6546b/packages/shared/src/constants/fields.ts#L40-L51

Reply:
Ah! Forget I said anything. Got this confused with the regular "TYPES" constant 🙂